File: solutions/cpp/raindrops/1/raindrops.cpp
#include<string>
#include "raindrops.h"

namespace raindrops {
    std::string convert(int number){
        std::string result = "";
        if(number == 1){
            result += std::to_string(number);
        }
        else if(number%3 == number%5 && number%3 == number%7){
            result += "PlingPlangPlong";
        }else if (number%3 == number%5 && number%5 == 0){
            result += "PlingPlang";
        }else if (number%3 == number%7 && number%3 == 0){
            result += "PlingPlong";
        }else if(number%5 == number%7 && number%5 == 0){
            result += "PlangPlong";
        }else if(number%3 == 0){
            result += "Pling";
        }else if (number%5 == 0){
            result += "Plang";
        }else if (number%7 == 0){
            result += "Plong";
        }else{
            result += std::to_string(number);
        }
        return result;
    }
    

}  // namespace raindrops

File: solutions/python/raindrops/1/raindrops.py
def convert(num):
    flag = False
    final_str = ""
    if num % 3 == 0:
        final_str += "Pling"
        flag = True
    if num % 5 == 0:
        final_str += "Plang"
        flag = True
    if num % 7 == 0:
        final_str += "Plong"
        flag = True

    if flag:
        return final_str
    return str(num)
